dev-libs/libical/libical-2.0.0~.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="An implementation of iCalendar protocols and data formats"
+DESCRIPTION="Libical is an Open Source implementation of the iCalendar \
+protocols and protocol data units. The iCalendar specification describes how \
+calendar clients can communicate with calendar servers so users can store \
+their calendar data and arrange meetings with other users."
+HOMEPAGE="http://libical.github.io/libical/"
+COPYRIGHT="Eric Busboom"
+LICENSE="GNU LGPL v2.1
+	MPL v1.0"
+REVISION="1"
+scrGitRev="1509e29de73044856271b18f1024acdee3d6f8ac"
+SOURCE_URI="https://github.com/libical/libical/archive/$scrGitRev.tar.gz"
+CHECKSUM_SHA256="3d4b02ff991916fee19badf6c0117627b6643fbdc1558e216e812108ab40fd9e"
+#PATCHES="calc-$portVersion.patchset"
+SOURCE_DIR="libical-$scrGitRev"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 ?x86"
+
+PROVIDES="
+	libical$secondaryArchSuffix = $portVersion
+	lib:libical$secondaryArchSuffix
+	lib:libical_cxx$secondaryArchSuffix
+	lib:libicalss$secondaryArchSuffix
+	lib:libicalss_cxx$secondaryArchSuffix
+	lib:libicalvcal$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	haiku${secondaryArchSuffix}_devel = $portVersion
+	devel:libical$secondaryArchSuffix
+	devel:libical_cxx$secondaryArchSuffix
+	devel:libicalss$secondaryArchSuffix
+	devel:libicalss_cxx$secondaryArchSuffix
+	devel:libicalvcal$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	libical$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:perl
+	"
+
+BUILD()
+{
+	cmake . \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-DCMAKE_INSTALL_LIBDIR=$libDir \
+		-DCMAKE_INSTALL_INCLUDEDIR=$incDir \
+		-DCMAKE_INSTALL_DATAROOTDIR=$dataDir
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	prepareInstalledDevelLibs libical libical_cxx libicalss libicalss_cxx libicalvcal
```

Comment:
This line is longer than 80 chars:
```
	prepareInstalledDevelLibs libical libical_cxx libicalss libicalss_cxx libicalvcal
```
So we need to split in at least 2 lines. Shall we split it with one lib per line, like this?
```
	prepareInstalledDevelLibs \
		libical \
		libical_cxx \
		libicalss \
		libicalss_cxx \
		libicalvcal
```